stubs/Pillow/METADATA.toml

```diff
@@ -1,6 +1,5 @@
-version = "10.0.*"
+version = "10.1.*"
 upstream_repository = "https://github.com/python-pillow/Pillow"
-partial_stub = true
```

Comment:
This was accidentally left in #10157 because it happened at the same time as #10114